src/unix/kqueue.c

```diff
@@ -34,6 +34,10 @@
 #include <fcntl.h>
 #include <time.h>
 
+#ifndef EV_OOBAND
+#define EV_OOBAND  EV_FLAG1
```

Comment:
EV_OOBAND exists in 10.7 ([ref](https://opensource.apple.com/source/xnu/xnu-1699.22.73/bsd/sys/event.h.auto.html)) so this doesn't need to be here.